packages/components/src/components/div100vh-container/div100vh-container.tsx

```diff
@@ -38,7 +38,7 @@ const Div100vhContainer = ({
         height: max_autoheight_offset ? '' : height_rule,
         maxHeight: max_autoheight_offset ? `calc(100rvh - ${max_autoheight_offset})` : '',
     };
-    if (is_bypassed) return children;
+    if (is_bypassed) return <React.Fragment>{children}</React.Fragment>;
```

Comment:
```suggestion
    if (is_bypassed) return children as JSX.Element;
```
Unfortunately, using React.Fragment is causing the component to work incorrectly which breaks UI (in responsive as I remember)

Reply:
Strange